docs/howto/krustlet-on-kubernetes-with-inlets.md

```diff
@@ -0,0 +1,273 @@
+# Running Krustlet on Kubernetes with inlets
+
+These are steps for running a krustlet node on your own computer. You can run any kind of 
+Kubernetes cluster you like, whether that's local on your computer or remote in managed 
+Kubernetes offering.
+
+These instructions are adapted from a [gist by Alex Ellis](https://gist.github.com/alexellis/d55d6d6a96ea9ae8d9d65b95297ec27e).
+
+## Prerequisites
+
+* [inlets - "The Cloud Native Tunnel"](https://docs.inlets.dev/)
```

Comment:
can you give a quick explanation what inlets does?

For example in the [AKS HOWTO](https://github.com/deislabs/krustlet/blob/master/docs/howto/krustlet-on-aks.md), it explains that we are about to boot a VM in the same resource group and virtual network as the original AKS cluster.

Reply:
It's a popular OSS tunnel (introduced in the intro) 

What kind of thing are you looking for?

Reply:
"
There are a number of ways to give the Kubernetes API server access to the krustlet's API. Various methods include using a VPN, a VM within the Kubernetes cluster's private network, or a tunnel. Inlets is an open source tunnel listed on the CNCF Landscape. It can be used to forward the port of the krustlet to the Kubernetes cluster so that the API server can access it as if it were deployed within the cluster directly."

Hope that's enough justification for you :)

Reply:
Looks great!